js/runner.go

```diff
@@ -262,6 +268,23 @@ func (r *Runner) newVU(idLocal, idGlobal uint64, samplesOut chan<- stats.SampleC
 	return vu, nil
 }
 
+// Will check if force http1 env variable has been set in order to force requests to be sent over h1
+// This feature is temporary until #936 is resolved
+func ForceHTTP1() bool {
```

Comment:
```suggestion
// forceHTTP1 checks if force http1 env variable has been set in order to force requests to be sent over h1
// TODO: This feature is temporary until #936 is resolved
func forceHTTP1() bool {
```
I think we can avoid exporting this function. As the linter is suggesting the comment must start with the function's name.